### PR TITLE
Introduce manifest for the JFrog CLI tool

### LINKED ISSUE #2539: [Package Request] JFrog CLI
### Package Name

JFrog CLI

### Package Component

Devops

### Package Description

JFrog CLI is a compact and smart client that provides a simple interface that automates access to Artifactory and Mission Control through their respective REST APIs.

### Project URL

https://github.com/jfrog/jfrog-cli

### PR DESCRIPTION
**Pull request by x80486**
Resolves #2539

### CODE REVIEW

File: blueprints/devops/jfrog-cli/ops2deb.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    - 2.59.1
+homepage: https://docs.jfrog-applications.jfrog.io/jfrog-applications/jfrog-cli
+summary: JFrog CLI is a client that provides a simple interface that automates access
+  to the JFrog products
+description: |-
+  JFrog CLI is a compact and smart client that provides a simple interface that
+  automates access to Artifactory and Mission Control through their respective
+  REST APIs. By using the JFrog CLI, you can greatly simplify your automation
+  scripts making them more readable and easier to maintain.
+fetch: https://releases.jfrog.io/artifactory/jfrog-cli/v2-jf/{{version}}/jfrog-cli-linux-{{arch}}/jf
```
Comment (x80486) [inline blueprints/devops/jfrog-cli/ops2deb.yml:15]: What is the difference between using `{{arch}}` or `{{goarch}}`?

Comment (fyhertz) [reply blueprints/devops/jfrog-cli/ops2deb.yml:15]: `goarch` simply maps debian archs to popular arch names in the go ecosystem: https://github.com/upciti/ops2deb/blob/main/src/ops2deb/jinja.py#L5

File: blueprints/devops/jfrog-cli/ops2deb.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+script:
+  - install --mode 0755 jf {{src}}/usr/bin/
```
Comment (fyhertz) [inline blueprints/devops/jfrog-cli/ops2deb.yml:17]: Please replace this with an install section:

```yaml
install:
  jf:/usr/bin
```

File: blueprints/devops/jfrog-cli/ops2deb.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+name: jfrog-cli
+matrix:
+  architectures:
+    - amd64
+  versions:
+    - 2.59.1
+homepage: https://docs.jfrog-applications.jfrog.io/jfrog-applications/jfrog-cli
+summary: JFrog CLI is a client that provides a simple interface that automates access
```
Comment (fyhertz) [inline blueprints/devops/jfrog-cli/ops2deb.yml:8]: No need to repeat the tool name: `client that provides a simple interface that automates access to JFrog products`

**Pull request merged**